### PR TITLE
feat/scss design added

### PR DESCRIPTION
**Pull request by mbilal-dev9**

### CODE REVIEW

File: index.scss [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  flex: 1;
+  padding: 40px;
+
+  h1 {
+    font-size: 18px;
+    color: $extra-light-blue-color;
+    margin: 0;
+  }
+
+  div {
```
Comment (umarabdullah23) [inline index.scss]: adding a nested div is not good unless its necessory, just add a classname
e.g. Sometimes we need to pick the first div, we then do: `>div`

Comment (umarabdullah23) [reply index.scss]: Also as this div has more nesting, just add a classname and get this div styling out of `description-section`

File: index.scss [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      height: 15px;
+      background-image: url(./assets/javascript.svg);
+      border-radius: 50%;
+      left: -2rem;
+      top: 2px;
+    }
+  }
+
+}
+
```
Comment (umarabdullah23) [inline index.scss:179]: same thing here too

one more idea, 


```
.login-section {
  padding: 40px;

  h1 {
    margin: 0;
    color: $yellow-color;
  }

  div {     //replace with some classname
    position: relative;
    margin: 8px 0px 0px;
    padding: 10px 0px 10px 0px;
    width: 40%;
    border: 2px solid;
    border-radius: 8px;
    font-size: 16px;
    background-color: $yellow-color;


  }


//nesting was unnecessory inside the upper div,so removed that

    input {
      border: none;
      outline: none;
      background-color: transparent;
      font-size: large;
      width: 75%;
      padding-left: 8px;
    }

    img {
      position: absolute;
      height: 40px;
      width: 40px;
      top: 2px;
      cursor: pointer;
    }
}
```

File: index.css [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+}
+.about-section li {
+  position: relative;
+}
+.about-section li:before {
+  position: absolute;
+  content: "";
+  width: 15px;
+  height: 15px;
+  background-image: url(./assets/javascript.svg);
```
Comment (umarabdullah23) [inline index.css:138]: for knowledge, if we add a data image then its very fast to load, so that image is preffered

**Review by umarabdullah23 — CHANGES_REQUESTED** [review]

File: index.css (hunk 3 above)
Comment (mbilal-dev9) [reply index.css:138]: can you give an example for data image. you means to say that an svg image?

**Pull request merged**